settings.js

```diff
@@ -62,6 +62,7 @@ window.spinnakerSettings = {
     // whether stages affecting infrastructure (like "Create Load Balancer") should be enabled or not
     infrastructureStages: infrastructureEnabled,
     jobs: false,
+    managedPipelineTemplatesV2: false,
```

Comment:
It might be worth adding an environment variable that can be used to enable the feature independently of halyard, similarly to the `templatesEnabled` var above.

Reply:
Good call, added!